src/widgets/CalendarWidget.js

```diff
@@ -70,7 +70,7 @@ export default class CalendarWidget extends InputWidget {
   loadZones() {
     const timezone = this.timezone;
     if (!zonesLoaded() && shouldLoadZones(timezone)) {
-      loadZones(timezone).then(() => this.emit('redraw'));
+      loadZones(timezone).then(() => this.rebuild());
```

Comment:
Did you actually test this?  ```this.rebuild``` is not a method in CalendarWidget, nor in InputWidget, no in Element.  I am pretty sure this will cause a JavaScript error. Please write a test to prove otherwise.

Reply:
Yeah, my mistake. I lost 'emit'.